### PR TITLE
Added german translations

### PR DESCRIPTION
**Pull request by DJaeger**

### CODE REVIEW

File: Spectacle/Resources/Localizations/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* The informative text of the alert displayed when prompting to restore Spectacle's default shortcuts */
+"AlertInformativeTextRestoreDefaults" = "Möchtest du die Standardtastenkombindationen wiederherstellen? Sämtliche eigenen Tastenkombinationen gehen dabei verloren.";
```
Comment (beatrupp) [inline Spectacle/Resources/Localizations/de.lproj/Localizable.strings]: Here's a typo `Standardtastenkombindationen`-> `Standardtastenkombinationen`

File: Spectacle/Resources/Localizations/de.lproj/Localizable.strings [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+"AlertInformativeTextShortcutAlreadyInUse" = "Die Tastenkombination \"%@\" wird bereits benutzt. Bitte wähle eine neue Tastenkombination.";
+
+/* The informative text of the alert displayed when a system-wide shortcut is already in use */
+"AlertInformativeTextSystemWideShortcutAlreadyUsed" = "Die Tastenkombination \"%@\" wird bereits benutzt. Bitte wähle eine neue Tastenkombination.\n        \n        Um diese Tastenkombination zu benutzen, ändere bitte die bestehende Kombination unter Tastatureinstellungen in den Systemeinstellungen";
+
+/* The message text of the alert displayed when prompting to restore Spectacle's default shortcuts */
+"AlertMessageTextRestoreDefaults" = "Dies wird die Standardtastenkombinationen wiederherstellen";
+
+/* The message text of the alert displayed when prompting to run Spectacle in the background */
+"AlertMessageTextRunningInBackground" = "Dies sorgt dafür das Spectacle im Hintergrund läuft.";
```
Comment (beatrupp) [inline Spectacle/Resources/Localizations/de.lproj/Localizable.strings]: `, dass`

File: Spectacle/Resources/Localizations/de.lproj/Spectacle.strings [hunk 3]
```diff
@@ -0,0 +1,93 @@
+/* Class = "NSMenuItem"; title = "Check for Updates..."; ObjectID = "2sj-vF-eSL"; */
```
Comment (beatrupp) [inline Spectacle/Resources/Localizations/de.lproj/Spectacle.strings]: I suggest to use the proper typography (ellipsis): `…`

File: Spectacle/Resources/Localizations/de.lproj/Spectacle.strings [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+"Isn-Og-AEv.title" = "Spectacle beenden";
+
+/* Class = "NSMenuItem"; title = "Left Half"; ObjectID = "Jae-vx-7ZC"; */
+"Jae-vx-7ZC.title" = "Linke Hälfte";
+
+/* Class = "NSMenu"; title = "Spectacle Status Item"; ObjectID = "KsA-8w-wcl"; */
+"KsA-8w-wcl.title" = "Spectacle Status Element";
+
+/* Class = "NSMenuItem"; title = "Preferences..."; ObjectID = "ME4-Br-HLT"; */
+"ME4-Br-HLT.title" = "Einstellungen...";
```
Comment (beatrupp) [inline Spectacle/Resources/Localizations/de.lproj/Spectacle.strings]: Use ellipsis `…`

File: Spectacle/Resources/Localizations/de.lproj/SpectaclePreferencesWindow.strings [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+"cEj-mH-kie.title" = "Bei Anmeldung öffnen";
+
+/* Class = "NSTextFieldCell"; title = "Lower Left:"; ObjectID = "dQP-TR-GMx"; */
+"dQP-TR-GMx.title" = "Unten links:";
+
+/* Class = "NSTextFieldCell"; title = "Shift"; ObjectID = "dww-8Q-sa1"; */
+"dww-8Q-sa1.title" = "Shift";
+
+/* Class = "NSTextFieldCell"; title = "Run..."; ObjectID = "eYU-TH-wcO"; */
+"eYU-TH-wcO.title" = "Ausführen...";
```
Comment (beatrupp) [inline Spectacle/Resources/Localizations/de.lproj/SpectaclePreferencesWindow.strings]: Use ellipsis `…`

**Comment by dotWee** [comment]
To match Apple's glossary, I would suggest to replace every `Tastenkombination/-en` with `Kurzbefehl/-e`. This is the word Apple uses to describe these kind of actions within the system preferences:

tastatureinstellungen

**Comment by linuxbandit** [comment]
duplicate of #809 ?

**Pull request closed without merging**